_overviews/scala3-book/fun-hofs.md

```diff
@@ -255,40 +255,40 @@ executeAndPrint(multiply, 3, 9)   // prints 27
 
 ## Function type signature consistency
 
-A great thing about learning about Scala’s function type signatures is that the syntax you use to define function input parameters is the same syntax you use to write anonymous functions and function variables.
+A great thing about learning about Scala’s function type signatures is that the syntax you use to define function input parameters is the same syntax you use to write functions literals.
```

Comment:
Change “functions” to “function” (without the ‘s’ at the end).

Reply:
Fixed